### PR TITLE
UnsupportedClassVersionError in test init

### LINKED ISSUE #24171: Test Failure: MPContextProp1_3_EE10_Test.initializationError UnsupportedClassVersionError: ContextServiceDefinition
Test Failure: com.ibm.ws.concurrent.mp.fat.jakarta.MPContextProp1_3_EE10_Test.initializationError

```
initializationError:java.lang.UnsupportedClassVersionError: JVMCFRE003 bad major version; class=jakarta/enterprise/concurrent/ContextServiceDefinition, offset=6
    at java.lang.ClassLoader.defineClassImpl(Native Method)
    at java.lang.ClassLoader.defineClassInternal(ClassLoader.java:397)
    at java.lang.ClassLoader.defineClass(ClassLoader.java:358)
    at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:154)
    at java.net.URLClassLoader.defineClass(URLClassLoader.java:730)
    at java.net.URLClassLoader.access$400(URLClassLoader.java:96)
    at java.net.URLClassLoader$ClassFinder.run(URLClassLoader.java:1187)
    at java.security.AccessController.doPrivileged(AccessController.java:774)
    at java.net.URLClassLoader.findClass(URLClassLoader.java:605)
    at java.lang.ClassLoader.loadClassHelper(ClassLoader.java:945)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:890)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:873)
    at java.lang.Class.forNameImpl(Native Method)
    at java.lang.Class.forName(Class.java:410)
    at java.lang.Class.getAnnotationCache(Class.java:2867)
    at java.lang.Class.getAnnotation(Class.java:2437)
    at componenttest.annotation.processor.TestServletProcessor.getQueryPath(TestServletProcessor.java:88)
    at componenttest.annotation.processor.TestServletProcessor.getServletTests(TestServletProcessor.java:67)
    at componenttest.custom.junit.runner.FATRunner.getChildren(FATRunner.java:162)
    at componenttest.custom.junit.runner.FATRunner.<init>(FATRunner.java:145)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:437)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:437)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:437)
```

This test bucket with tests for both Jakarta EE 9 and Jakarta EE 10 should skip the Jakarta EE 10 tests when running on Java SE 8 but is encountering this error instead, likely when trying to load the imports of MPContextProp1_3_EE10_Test before identifying its `@MinimumJavaLevel(javaLevel = 11)` annotation,

```
import concurrent.mp.fat.v13.ee10.web.MPContextProp1_3_EE10_TestServlet;

@RunWith(FATRunner.class)
@MinimumJavaLevel(javaLevel = 11)
public class MPContextProp1_3_EE10_Test extends FATServletClient {
```

We can work around this by refactoring to avoid the import.

### PR DESCRIPTION
**Pull request by njr-11**
fixes #24171

### CODE REVIEW

File: dev/com.ibm.ws.concurrent.mp_fat_jakarta/fat/src/com/ibm/ws/concurrent/mp/fat/jakarta/MPContextProp1_3_EE10_Test.java [hunk 1]
```diff
@@ -16,28 +16,24 @@
...
 import componenttest.custom.junit.runner.FATRunner;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.FATServletClient;
-import concurrent.mp.fat.v13.ee10.web.MPContextProp1_3_EE10_TestServlet;
 
-// TODO This temporarily runs with MP Context Propagation 1.3 because 2.0 isn't available yet.
-//      See the comments in FATSuite.java
 @RunWith(FATRunner.class)
 @MinimumJavaLevel(javaLevel = 11)
 public class MPContextProp1_3_EE10_Test extends FATServletClient {
```
Comment (KyleAure) [inline dev/com.ibm.ws.concurrent.mp_fat_jakarta/fat/src/com/ibm/ws/concurrent/mp/fat/jakarta/MPContextProp1_3_EE10_Test.java:32]: I think the issue here is that even though the test class has the `@MinimumJavaLevel` annotation junit processes tests by creating a list of test methods, and then filtering them out. Our custom FATRunner uses the `@TestServlet` annotation to grab the servlet class and find additional test methods. 

I did some digging and I think the solution would be to add the `@MinimumJavaLevel(javaLevel = 11)` to the servlet class which also contains `@Test` annotations. 

Your solution also works, but I think the additional annotation change might be the actual solution here.  This obviously doesn't work if two test classes share the same servlet, and that might be a situation for the build team to look at.

**Review by KyleAure — CHANGES_REQUESTED** [review]

**Review by KyleAure — APPROVED** [review]
Approved, I think this is actually happening when creating a list of methods.  My previous suggestion won't work. This fix looks good.

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build

**Pull request merged**